src/index.html

```diff
@@ -1,22 +1,252 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Landing Page</title>
+    <meta name="format-detection" content="telephone=no">
+    <title>NOTHING</title>
     <link
       rel="stylesheet"
       href="./styles/main.scss"
     />
+    <link rel="icon" href="./styles/images/favicon.png"
+      type="image/png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@700&family=Space+Mono&display=swap" rel="stylesheet">
     <script
       src="scripts/main.js"
       defer
     ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="top-bar">
+        <a href="/layout_landing-page" class="top-bar__logo-link">
+          <img src="./styles/images/logo-2x.png" alt="Nothing Company logo"
+          class="top-bar__logo" />
+        </a>
+
+        <div class="top-bar__icons">
+          <a href="tel:+1 234 5555-55-55" class="icon icon--phone"></a>
+          <a href="#menu" class="icon icon--burger-menu"></a>
+        </div>
+
+      </div>
+
+      <div class="header__bottom">
+        <h1 class="header__main-text">Bring joy back<br>to the everyday</h1>
+      </div>
+    </header>
+    <aside class="menu page__menu" id="menu">
+
+      <div class="top-bar">
+        <a href="/layout_landing-page" class="top-bar__logo--link">
+          <img src="./styles/images/logo-2x.png" alt="Nothing Company logo"
+                  class="top-bar__logo" />
+        </a>
+        <div class="top-bar__icons">
+          <a href="#" class="icon icon--close"></a>
+        </div>
+      </div>
+
+      <div class="menu__content">
+        <nav class="nav menu__nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link" href="#products">Products</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#store">Store</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="/">Login</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="/">Cart(0)</a>
+            </li>
```

Comment:
When I click on it, your project crashes(For all such cases)
```suggestion
            <li class="nav__item">
              <a class="nav__link" href="#">Login</a>
            </li>
            <li class="nav__item">
              <a class="nav__link" href="#">Cart(0)</a>
            </li>
```